### DISCUSSION TITLE
What happened to the "Upload to YouTube" option?

### DISCUSSION THREAD

**Discussion by masthem** [TheAliceProject/alice3#281] [Q&A] [answered]
I used the Upload to YouTube tool to easily record my projects and save them. I didn't really mind the quality or format. But now that I downloaded Alice 3 on my home computer the button is gone. Why is it gone now?
I am currently on Alice 3.7.0.0 Build 876.

**Comment by dabeshouse** [comment, marked as answer]
The upload feature was gaining bugs and problems faster than the small Alice team could address them and we recognized that recording video is done better by many other tools.

We have a [help page with a video](https://www.alice.org/resources/how-tos/recording-and-sharing-alice-worlds/) explaining how to use one possible tool, [OBS](https://obsproject.com/).

**Reply by masthem** [reply]
Oh alright, that’s good. Thank you!
